clients/src/test/java/org/apache/kafka/common/utils/MockTime.java

```diff
@@ -73,4 +73,8 @@ public void sleep(long ms) {
         highResTimeNs.addAndGet(TimeUnit.MILLISECONDS.toNanos(ms));
     }
 
+    public void setCurrentTimeMs(long ms) {
```

Comment:
I am just wondering, if one could break things if time gets set back to the past? Should we allow for this or guard against it?

Reply:
Ack.